packages/api-explorer/src/block-types/Content.jsx

```diff
@@ -1,119 +1,9 @@
-const CallOut = require('./CallOut');
-const Html = require('./Html');
-const TextArea = require('./TextArea');
-const Code = require('./Code');
-const ImageBlock = require('./Image');
-const Embed = require('./Embed');
-const Parameters = require('./Parameters');
-const ApiHeader = require('./ApiHeader');
-
 const React = require('react');
-const PropTypes = require('prop-types');
-
-const parseBlocks = require('../lib/parse-magic-blocks');
-
-const Loop = ({ content, column, flags }) => {
-  const elements = content.map((block, key) => {
-    const props = { key, block, flags };
-    switch (block.type) {
-      case 'textarea':
-        // eslint-disable-next-line react/no-array-index-key
-        return <TextArea {...props} />;
-      case 'html':
-        // eslint-disable-next-line react/no-array-index-key
-        return <Html {...props} />;
-      case 'embed':
-        // eslint-disable-next-line react/no-array-index-key
-        return <Embed {...props} />;
-      case 'api-header':
-        // eslint-disable-next-line react/no-array-index-key
-        return <ApiHeader {...props} />;
-      case 'code':
-        // eslint-disable-next-line react/no-array-index-key
-        return <Code {...props} dark={column === 'right'} />;
-      case 'callout':
-        // eslint-disable-next-line react/no-array-index-key
-        return <CallOut {...props} />;
-      case 'parameters':
-        // eslint-disable-next-line react/no-array-index-key
-        return <Parameters {...props} />;
-      case 'image':
-        // eslint-disable-next-line react/no-array-index-key
-        return <ImageBlock {...props} />;
-      default:
-        return null;
-    }
-  });
-  return <div>{elements}</div>;
-};
-
-const Content = props => {
-  const { body, isThreeColumn } = props;
-  const content = parseBlocks(body);
-
-  const left = [];
-  const right = [];
-  content.forEach(elem => {
-    if (elem.sidebar) {
-      right.push(elem);
-    } else {
-      left.push(elem);
-    }
-  });
-
-  if (isThreeColumn === true) {
-    return (
-      <div className="hub-reference-section">
```

Comment:
What did we end up doing with this requirement? This is definitely something that we still have to handle!

Reply:
Very good point... still exploring the best approach here. Open to any suggestions!